_posts/2015-06-12-topics.md

```diff
@@ -871,3 +878,238 @@ This topic is deprecated. Please use [shipment:packed](#shipment_packed) instead
     }
 }
 ```
+<h3 id="tracker_updated">
+    tracker:updated
+</h3>
+
+```json
+{
+    "topic" : "tracker:updated",
+    "message" : {
+        "order_ref" : "9099",
+        "order_unique_id" : "100000018",
+        "shipment_unique_id" : "100000007",
+        "warehouse_id": 1,
+        "package_id" : 99,
+        "source" : "shopify",
+        "carrier" : "fedex",
+        "service" : "FEDEX_2_DAY",
+        "number" : "794657040105",
+        "url" : "https://www.fedex.com/apps/fedextrack/?action=track&tracknumbers=794657040105",
+        "webhook_increment_id" : 1,
+        "event_timestamp" : "2021-12-07T16:08:15+00:00"
+    }
```

Comment:
`webhook_increment_id` and `event_timestamp` actually appear outside the `message` object in all cases.